tests/writer.rs

```diff
@@ -275,6 +275,7 @@ mod tests {
             datatype: NiftiType::Rgb24 as i16,
             pixdim: [1.0; 8],
             sform_code: 2,
+            qform_code: 0,
             srow_x: [1.0, 0.0, 0.0, 0.0],
             srow_y: [0.0, 1.0, 0.0, 0.0],
             srow_z: [0.0, 0.0, 1.0, 0.0],
```

Comment:
You can remove `pixdim` and `srow_#` on the three tests. And maybe add a `get_rgb_header` function to avoid repeating those 7 lines.